mediabridge/main.py

```diff
@@ -1,4 +1,32 @@
+import logging
+
+import typer as typer
+from tqdm.contrib.logging import logging_redirect_tqdm
+
 from mediabridge.data_processing import wiki_to_netflix
 
-q = wiki_to_netflix.format_sparql_query("The Room", 2003)
-print(q)
+
+def main(verbose: bool = typer.Option(False, "--verbose", "-v"), test: bool = False):
+    if test:
+        # log all messages to file
+        logging.basicConfig(
+            level=logging.DEBUG,
+            filename="out/mediabridge.log",
+            filemode="a",
+            format="%(asctime)s,%(msecs)d %(name)s %(levelname)s %(message)s",
+            datefmt="%H:%M:%S",
+        )
+        wiki_to_netflix.process_data(True)
+        return
+
+    if verbose:
+        logging.basicConfig(level=logging.INFO)
+
```

Comment:
Any consideration for logging in "production"?

Reply:
Sorry for the wait. The --test option (logging all messages to a file) is how I think the production system will log, but I'm sure we can swap the logging method for whatever monitoring system we want to use later.